CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -705,7 +705,7 @@ public function postProcess() {
               }
               else {
                 CRM_Core_DAO::setFieldValue('CRM_Price_DAO_PriceField', $priceFieldID, 'is_required', 0);
-                CRM_Core_DAO::setFieldValue('CRM_Price_DAO_PriceFieldValue', $priceFieldValueID, 'label', 'Other Amount');
+                CRM_Core_DAO::setFieldValue('CRM_Price_DAO_PriceFieldValue', $priceFieldValueID, 'label', ts('Other Amount'));
```

Comment:
@mlutfy in DB value is saved as 'Other Amount' so I think ts('Other Amount') should be changed to 'Other Amount'